prog/minio/minio_server_nexus.rb

```diff
@@ -158,4 +168,21 @@ def available?
   rescue
     false
   end
+
+  def create_certificate
+    root_cert = OpenSSL::X509::Certificate.new(minio_server.cluster.root_cert_1)
+    root_cert_key = OpenSSL::PKey::EC.new(minio_server.cluster.root_cert_key_1)
+    if root_cert.not_after < Time.now + 60 * 60 * 24 * 365 * 1
+      root_cert = OpenSSL::X509::Certificate.new(minio_server.cluster.root_cert_2)
+      root_cert_key = OpenSSL::PKey::EC.new(minio_server.cluster.root_cert_key_2)
+    end
+
+    Util.create_certificate(
+      subject: "/C=US/O=Ubicloud/CN=#{minio_server.cluster.ubid} Server Certificate",
+      extensions: ["subjectAltName=DNS:#{minio_server.cluster.hostname},DNS:#{minio_server.hostname},IP:#{minio_server.vm.ephemeral_net4}", "keyUsage=digitalSignature,keyEncipherment", "subjectKeyIdentifier=hash", "extendedKeyUsage=serverAuth"],
```

Comment:
Why do we put both cluster and server hostname here?

Reply:
minio servers connect to each other via server hostname that is in the /etc/hosts not part of the dns